ivy/functional/frontends/tensorflow/tensor.py

```diff
@@ -125,5 +125,12 @@ def __sub__(self, y, name="sub"):
     def __truediv__(self, y, name="truediv"):
         return y.__rtruediv__(self.data)
 
+    def __len__(self):
+        raise ivy.exceptions.IvyError(
+            f"len is not well defined for a symbolic Tensor "
+            f"(*tensor name*). Please call `x.shape` rather than "
+            f"`len(x)` for shape information."
+        )
```

Comment:
Since we haven't included `self.name`, you could print this message instead:
"len is not well defined for symbolic Tensors. "
"Please call `x.shape` rather than `len(x)` for "
"shape information."

Reply:
Okay sure, makes sense! I wanted to match exactly the tf behavior. Will change that